### PR TITLE
perf(agw): Remove unused timer serialization functions

### LINKED ISSUE #9703: chore(mme): Clean up dead code
MME has some unused code that needs clean-up:

- MME/NAS timer id serialization/deserialization
- Gtpnl data path
- ASN release 8,9 and 10
- MME state converter hash tables
- Unused code in lib/bstr/

### PR DESCRIPTION
**Pull request by ssanadhya**
Signed-off-by: GitHub <noreply@github.com>

## Summary

Timer ids are not persistent across service restart, so no need to serialize them and save then in state. This change marks the Timer fields in UE context proto as deprecated. It also removes the serialization/deserialization functions in MME state converter

## Test Plan

S1ap integration tests

### CODE REVIEW

**Comment by ulaskozat** [comment]
@ssanadhya could you add such BE activities under https://app.zenhub.com/workspaces/magma--better-engineering-5f84ca9668d267002021e827/issues/magma/magma/6663

**Review by ardzoht — APPROVED** [review]
lgtm

File: lte/gateway/c/core/oai/tasks/mme_app/mme_app_state_converter.cpp [hunk 1]
```diff
@@ -722,24 +693,18 @@ void MmeNasStateConverter::proto_to_ue_mm_context(
...
-      ue_context_proto.ulr_response_timer(),
-      &state_ue_mm_context->ulr_response_timer);
-  proto_to_mme_app_timer(
-      ue_context_proto.paging_response_timer(),
-      &state_ue_mm_context->paging_response_timer);
+  // Initialize timers to INVALID IDs
+  state_ue_mm_context->mobile_reachability_timer.id = MME_APP_TIMER_INACTIVE_ID;
+  state_ue_mm_context->implicit_detach_timer.id     = MME_APP_TIMER_INACTIVE_ID;
+  state_ue_mm_context->initial_context_setup_rsp_timer = (nas_timer_t){
+      MME_APP_TIMER_INACTIVE_ID, MME_APP_INITIAL_CONTEXT_SETUP_RSP_TIMER_VALUE};
```
Comment (ulaskozat) [inline lte/gateway/c/core/oai/tasks/mme_app/mme_app_state_converter.cpp]: mme_config.nas_config.tics_msec

Comment (ulaskozat) [reply lte/gateway/c/core/oai/tasks/mme_app/mme_app_state_converter.cpp]: can you also fix https://github.com/magma/magma/blob/1dffd37504977682fc185e28dee7bde592b9717d/lte/gateway/c/core/oai/tasks/mme_app/mme_app_context.c#L141 as it is incorrectly setting it as mme_config.nas_config.ts6a_msec

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**